### PR TITLE
New sample type and group link sci 701

### PR DESCRIPTION
**Pull request by mz3944**
https://biosistemika.atlassian.net/browse/SCI-701

### CODE REVIEW

**Comment by mlorb** [comment]
In the Epic document (Epic026) - Image(5) is displayed where the user should be redirected, when he clicks on the link (an empty entry field should appear, into which user can enter a new Sample type/group).
No second click for creating type/group should be needed.

File: app/assets/javascripts/sitewide/utils.js [hunk 1]
```diff
@@ -215,3 +215,30 @@ function initPageTutorialSteps(pageFirstStepN, pageLastStepN, nextPagePath,
...
+ * @param  {number} selectedIdx Index of element to be selected
+ * @return {Object}             This
+ */
+$.fn.makeDropdownOptionsLinks = function(selectedIdx) {
+  selectedIdx = _.isUndefined(selectedIdx) ? 1 : selectedIdx;
+  $(this).change(function() {
+    location = $(this).find('option:selected').attr('href');
+  });
+
+  var dropdownEl = $(this).parent().find('.dropdown-menu .inner li');
```
Comment (mz3944) [inline app/assets/javascripts/sitewide/utils.js]: Ah, forgot to delete this one!

File: app/assets/javascripts/samples/sample_datatable.js [hunk 2]
```diff
@@ -762,11 +769,15 @@ function createSampleTypeSelect(data, selected) {
...
+  selected = _.isUndefined(selected) ? 1 : selected + 1;
+
   var $selectGroup = $('<select></select>')
     .attr('name', 'sample_group_id').addClass('show-tick');
 
-  var $span = $("<span></span>").addClass('glyphicon glyphicon-asterisk');
-  var $option = $('<option></option>')
+  var $option = $("<option href='/organizations/1/sample_groups'></option>")
+                  .text(I18n.t('samples.table.add_sample_group'));
+  $selectGroup.append($option);
```
Comment (mz3944) [inline app/assets/javascripts/samples/sample_datatable.js]: No, we are appending options to select HTML tag.

File: app/assets/javascripts/samples/sample_datatable.js [hunk 3]
```diff
@@ -736,14 +736,21 @@ function createTdElement(content) {
...
+ * @return {Object} select dropdown
  */
 function createSampleTypeSelect(data, selected) {
+  selected = _.isUndefined(selected) ? 1 : selected + 1;
+
   var $selectType = $('<select></select>')
     .attr('name', 'sample_type_id').addClass('show-tick');
 
-  var $option = $('<option></option>')
+  var $option = $("<option href='/organizations/1/sample_types'></option>")
```
Comment (ZmagoD) [inline app/assets/javascripts/samples/sample_datatable.js]: Do you hardcoded the link that links always to the first organization ??????? svašta

**Pull request merged**